Update Redux-Toolkit Docs for Integration (#1323)
infinitered/reactotron: docs/plugin-redux.md

  
+ export default reactotron
```

> ## Using Redux-toolkit
> Then, add enhancer from `Reactotron.createEnhancer()` to `enhancers` in `configureStore`
> 
> ```diff
> import {configureStore} from '@reduxjs/toolkit';
> + import Reactotron from './ReactotronConfig'
> const store = configureStore({
>    ... (reducers, middlewares etc),
> +  enhancers: __DEV__ ? [Reactotron.createEnhancer!()] : [],
> });
> ```
> 
> ## Using Classic Redux
> Then, add enhancer from `Reactotron.createEnhancer()` to `createStore` as last parameter

Reviewer: ```suggestion
## Using Redux Toolkit (RTK)

If you're using RTK, add the enhancer from `Reactotron.createEnhancer()` to `enhancers` in `configureStore`

```diff
import {configureStore} from '@reduxjs/toolkit';
+ import Reactotron from './ReactotronConfig'
const store = configureStore({
   ... (reducers, middlewares etc),
+  enhancers: __DEV__ ? [Reactotron.createEnhancer!()] : [],
});
```

## Using Classic Redux

If you're not using RTK, add the enhancer from `Reactotron.createEnhancer()` to `createStore` as the last parameter
```
